e2e/rollup/src/rollup.test.ts

```diff
@@ -235,6 +235,64 @@ export default config;
   });
 
   describe('useLegacyTypescriptPlugin', () => {
+    it('should use @rollup/plugin-typescript by default when useLegacyTypescriptPlugin is not specified', async () => {
```

Comment:
nit/question: Do we need this test case?

I suggest not adding an extra e2e test case (which takes time to run) to cover the default value, which we can't explicitly check for the plugin usage, and we already check the build of a similar setup in other test cases.

Reply:
We can remove it.